src/Uno.UI/UI/Xaml/Markup/Reader/XamlConstants.cs

```diff
@@ -12,7 +12,7 @@ internal static class XamlConstants
 		public const string BundleResourcePrefix = "ms-appx:///";
 
 		public const string RootFoundationNamespace = "Windows.Foundation";
-		public const string RootWUINamespace = "Windows.UI";
+		public const string RootWUINamespace = "Windows" + ".UI"; // Keep split for the WinUI conversion tool
```

Comment:
This one won't get replaced by the tool, and the below will be replaced, right?

Just to make sure I understand the change here.

Reply:
That's correct. It's a mirror of this file: https://github.com/unoplatform/uno/blob/927a09f70d90f6fb1c9f03a3b77f5577d57240fe/src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlConstants.cs